submissions/lillianNwaoha/css/signup.css

```diff
@@ -0,0 +1,320 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+  overflow-x: hidden;
+}
+nav {
+  background-color: #0066f5;
+  height: 190px;
+  width: auto;
+}
+.logo img {
+  width: 90px;
+  height: 90px;
+  margin-left: 20px;
+  margin-top: 15px;
+}
+.nav-link {
+  list-style: none;
+  margin-left: -8px;
+  margin-right: -50px;
+}
+.nav-link a {
+  text-decoration: none;
+  color: white;
+  font-size: 14px;
+  padding-bottom: 10px;
+  display: block;
+  margin-left: 40px;
+}
+.top {
+  text-align: center;
+  width: 50%;
+  margin-left: 125px;
+  margin-top: 20px;
+}
+.top h1 {
+  color: #0066f5;
+  font-size: 18px;
+  width: 50%;
+  margin-left: 55px;
+}
+.top p {
+  font-size: 14px;
+}
+.form p {
+  margin-top: 5px;
+  font-size: 14px;
+  text-align: center;
+  margin-left: 60px;
+  padding-right: -10px;
+  padding-left: -30px;
+  width: 70%;
+}
+
+.forms {
+  margin: 40px 0px 0px 20px;
+  border-radius: 10px;
+}
+fieldset {
+  border: 1px solid black;
+  border-radius: 10px;
+  height: 690px;
+  width: 70%;
+  margin-left: 70px;
+  margin-top: 30px;
+  padding-top: 30px;
+  margin-bottom: 30px;
+  padding-right: -10px;
+}
+label {
+  margin-left: 30px;
+  padding-top: 30px;
+  margin-top: 70px;
+  font-weight: lighter;
+}
+input {
+  margin-left: 30px;
+  border-radius: 6px;
+  width: 70%;
+  height: 6%;
+  margin-bottom: 20px;
+}
+input[type='checkbox'] {
+  width: 20px;
+  height: 20px;
+  margin-top: 10px;
+}
+label[for='checkbox'] {
+  color: #0066f5;
+  font-size: 14px;
+  margin-left: 10px;
+}
+fieldset > button {
+  margin-left: 120px;
+  padding: 10px 20px;
+  background-color: #0066f5;
+  color: white;
+  border-radius: 6px;
+  border: none;
+}
+.footer {
+  background-color: #0066f5;
+  padding-top: 50px;
+  width: auto;
+}
+.foot {
+  display: block;
+  margin: 0px 60px 0px 30px;
+}
+.footer-link {
+  margin-left: 20px;
+  list-style: none;
+  margin-top: 2px;
+}
+.footer-link a {
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: normal;
```

Comment:
For everywhere you've used normal as the value of an attribute, you should remove it. It is normal by default. @Lillian-1